### PR TITLE
Update lang ja string

### PR DESCRIPTION
**Pull request by kenjis**
#148 and some little fixes.

```
$ ./phpunit --filter JapaneseTranslationTest
PHPUnit 9.5.0 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.4.13
Configuration: /Users/kenji/work/codeigniter/codeigniter4-translations/phpunit.xml.dist

...............................                                   31 / 31 (100%)

Time: 00:00.029, Memory: 14.00 MB

OK (31 tests, 31 assertions)
```

### CODE REVIEW

File: Language/ja/CLI.php [hunk 1]
```diff
@@ -11,10 +11,20 @@
...
 return [
-	'commandNotFound' => '"{0}"というコマンドはありません。', //Command "{0}" not found.
+	'commandNotFound' => '"{0}" というコマンドはありません。', //Command "{0}" not found.
+	'altCommandSingular'  => 'これですか？', //Did you mean this?
+	'altCommandPlural'    => 'この中の1つですか？', //Did you mean one of these?
 	'helpUsage'       => '使用法:', //Usage:
 	'helpDescription' => '説明:', //Description:
 	'helpOptions'     => 'オプション:', //Options:
 	'helpArguments'   => '引数:', //Arguments:
 	'invalidColor'    => '「{1}」は無効な {0} カラーです。', //Invalid {0} color: {1}.
```
Comment (paulbalandan) [inline Language/ja/CLI.php:21]: Can you fix the alignment of `=>` here?

File: Language/ja/HTTP.php [hunk 2]
```diff
@@ -70,4 +70,7 @@
 	'uploadErrNoTmpDir'  => 'ファイルをアップロードできませんでした。: 一時ディレクトリが見つかりません。', //File could not be uploaded: missing temporary directory.
 	'uploadErrExtension' => 'ファイルアップロードはPHP機能拡張により停止されました。', //File upload was stopped by a PHP extension.
 	'uploadErrUnknown'   => 'ファイル "%s" は未知のエラーのためアップロードされませんでした。', //The file "%s" was not uploaded due to an unknown error.
+
+	// SameSite setting
+	'invalidSameSiteSetting'     => 'SameSite 設定には None、Lax、Strict または空文字列を指定する必要がありますが、「{0}」が指定されました。', //The SameSite setting must be None, Lax, Strict, or a blank string. Given: {0}
```
Comment (paulbalandan) [inline Language/ja/HTTP.php:75]:
```suggestion
	'invalidSameSiteSetting' => 'SameSite 設定には None、Lax、Strict または空文字列を指定する必要がありますが、「{0}」が指定されました。', //The SameSite setting must be None, Lax, Strict, or a blank string. Given: {0}
```

File: Language/ja/Migrations.php [hunk 3]
```diff
@@ -44,9 +44,15 @@
 	'on'                => '移行先: ', //Migrated On:
 	'migSeeder'         => 'seeder名', //Seeder name
 	'migMissingSeeder'  => 'seeder名が必要です。', //You must provide a seeder name.
+	'nameSeeder'        => 'seederファイル名', //Name the seeder file
 	'removed'           => 'ロールバック: ', //Rolling back:
 	'added'             => '実行中: ', //Running:
 
-	'version'  => 'バージョン', //Version
+	// Migrate Status
+	'namespace'         => '名前空間', //Namespace
```
Comment (paulbalandan) [inline Language/ja/Migrations.php:52]:
```suggestion
	'namespace' => '名前空間', //Namespace
```

File: Language/ja/Migrations.php [hunk 4]
```diff
@@ -44,9 +44,15 @@
...
 	'filename' => 'ファイル名', //Filename
+	'version'  => 'バージョン', //Version
+	'group'             => 'グループ', //Group
+	'on'                => '移行先: ', //Migrated On:
+	'batch'             => 'バッチ', //Batch
```
Comment (paulbalandan) [inline Language/ja/Migrations.php:57]: Please align with the corrected alignment of `namespace` above.

**Review by paulbalandan — COMMENTED** [review]
Some fixes for CS.

Also, since the maintainer of ja has no activity for 4 months, can you update your name in the README?

**Comment by paulbalandan** [comment]
Also, please rebase so that we can see the fixed tests running on this.

**Comment by kenjis** [comment]
@paulbalandan I've done all of your suggestions. Is this okay?

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**